TACC/Core-Portal
# task/FP-1922: Increase `CustomMessageTemplate` `max_length` to 1000 characters

## Overview

Updating the CustomMessageTemplate template to increase character limit for APCD's long messages

## Related

* [FP-1922](https://jira.tacc.utexas.edu/browse/FP-1922)

## Changes
Simple change to the `message` field `max_length` attribute


## Testing

1. Go to core.test/core/admin/portal_messages/custommessagetemplate/
2. Add a custom template message to a page you can access (dashboard is a good test)
3. Add more than 200 characters in the message field.
4. Check the dashboard to make sure the entire message displays.
5. Go back to core.test/core/admin/portal_messages/custommessagetemplate/
6. Edit your previous message and add more than 1000 characters.
7. Make sure the message fails.

## UI



## Notes
